app.py

```diff
@@ -292,9 +292,10 @@ def get(self, blobres):
     pcapdata = PcapData.GetOrInsertFromBlob(blob_info)
 
     self.response.headers['Content-Type'] = 'application/json'
+    # include in js_bundle any content to be included in json being passed to index.html
     js_bundle = {
         'js_packets': pcapdata.js_packets,
-        'js_streams': pcapdata.js_streams,
```

Comment:
there's actually a good reason to leave this extra trailing comma: it means that if you add an extra entry anywhere in the dict, including the end, then the diff will only change the new line, it won't require a change to any other semantically-unrelated lines.  This helps reduce merge conflicts when multiple people are editing at the same time.  (Sadly, in javascript, the trailing comma is considered a syntax error in IE so we can't do it.  But it works in pretty much every other language and is a good idea.)